pages/common/tldr.md

```diff
@@ -7,14 +7,14 @@
 
 `tldr {{command}}`
 
-- Show the tldr page for `cd`, overriding the default platform:
+- Show the tldr page for `cd`, overriding the default platform(if the client supports -p):
```

Comment:
The Ubuntu haskell client does not

Reply:
0.4 of https://github.com/psibi/tldr-hs doesn't, but 0.9 (the latest version) does, and we're working on that (#5962)

Also, [`-p` is part of the client spec](https://github.com/tldr-pages/tldr/blob/ddf755f35520ab3b128255edd5da32a85f15bd02/CLIENT-SPECIFICATION.md#arguments), so we shouldn't have to do this.

Reply:
Ok, will do. 
Is their an easier way to do it than doing a commit?

Reply:
Well, you can add a suggestion and apply it right here...
```suggestion
- Show the tldr page for `cd`, overriding the default platform:
```